### PR TITLE
Add 7 missing implicit quad mnemonics to 45GS02

### PR DESCRIPTION
**Pull request by mlund**
- [x] Add missing implicit mnemonics, e.g. `DEQ`, `INQ`, etc.
- [x] Rename `CPQ` to `CMPQ` to follow the MEGA65 book. The ACME `CPQ` syntax is is now an alias to `CMPQ` and thus still supported.
- [x] Reorder `all-45gs02opcodes.s` so that mnemonics are grouped by name. This makes it easier to compare with the MEGA65 book.
- [x] Remove unsupported mnemonics that was present in ACME but their use is currently undefined.
- [x] (Dis)assemble roundtrip OK

### CODE REVIEW

**Comment by asiekierka** [comment]
It might be worth considering aliasing `CPQ` to `CMPQ`. Not sure, though.

**Comment by mlund** [comment]
Alias added.

File: llvm/lib/Target/MOS/MOSInstrInfo.td [hunk 1]
```diff
@@ -500,6 +500,15 @@ let Predicates = [Has45GS02] in {
 
 let DecoderNamespace = "45gs02" in {
 
+/// Q seudo register implicit (NEG NEG predix, 3 bytes)
```
Comment (mysterymath) [inline llvm/lib/Target/MOS/MOSInstrInfo.td:503]: nit: "pseudo", "prefix"

**Review by mysterymath — APPROVED** [review]

File: llvm/lib/Target/MOS/MOSInstrInfo.td (hunk 1 above)
Comment (mlund) [reply llvm/lib/Target/MOS/MOSInstrInfo.td]: Fixed!

**Pull request merged**